### PR TITLE
Palak Assignment 8 Done

### PR DESCRIPTION
**Pull request by PalakSJun20**

### CODE REVIEW

File: src/palakSJun20/BankingSystem.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+5) allTransactionSummary() method should show by both users total how many times debit, credit & printBalance method get called.
+Hint : Create 3 static variable will be required.
+Output:
+All transaction summary : Credit - 7 times, Debit - 3 times, printBalance - 1 time*/
+package palakSJun20;
+
+public class BankingSystem {
+	String customerName;
+	int customerCredit , customerDebit , customerBalance;
```
Comment (KrishnaKTechnocredits) [inline src/palakSJun20/BankingSystem.java]: Remove these unused variables.

File: src/palakSJun20/BankingSystem.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	static int creditCount, debitCount, printBalanceCount;
+	
+	void setCustomerName(String customerName){
+		this.customerName = customerName;
+	}
+	
+	int debitTransaction(int debitAmount){
+		customerDebitCount++;
+		debitCount++;
+		return this.customerBalance = customerBalance - debitAmount;
```
Comment (KrishnaKTechnocredits) [inline src/palakSJun20/BankingSystem.java]: You should write the condition to verify the insufficient balance.

File: src/palakSJun20/BankingSystem.java (hunk 1 above)
Comment (PalakSJun20) [reply src/palakSJun20/BankingSystem.java]: Done, Removed unused Variable

File: src/palakSJun20/BankingSystem.java (hunk 2 above)
Comment (PalakSJun20) [reply src/palakSJun20/BankingSystem.java]: Added condition

**Review by KrishnaKTechnocredits — COMMENTED** [review]
Code looks good to me.

**Pull request merged**